Sources/pointfreeco/main.swift

```diff
@@ -57,6 +58,8 @@ print("Connected!")
 
 let router = Router()
 
+router.all(middleware: Compression())
```

Comment:
this has to be added before our router middleware. the reason is that what they call "middleware" is more like our "middleware transformers", and as we know... they compose backwards!